jax/random.py

```diff
@@ -14,10 +14,30 @@
 
 """JAX pseudo-random number generators (PRNGs).
 
-The JAX PRNG system is based on "Parallel random numbers: as easy as 1, 2, 3"
-(Salmon et al. 2011). For details on the design and its motivation, see:
-
-https://github.com/google/jax/blob/master/design_notes/prng.md
+Usage:
+
+```python
+rng = jax.random.PRNGKey(seed)
+for i in xrange(num_steps):
+  rng, rng_input = jax.random.split(rng)
+  params = compiled_update(rng_input, params, next(batches))
+```
+
+Context:
+
+Among other requirements, Jax PRNG aims at:
```

Comment:
Among other requirements, the JAX PRNG aims to:

Reply:
Done.